lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2287,14 +2287,14 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
             }
             self.file._annotations[-1][1].append(link_annotation)
 
-        # If fonttype != 3 or there are no multibyte characters, emit the whole
-        # string at once.
-        if fonttype != 3 or all(ord(char) <= 255 for char in s):
+        # If fonttype != 3 emit the whole string at once without manual
```

Comment:
possibly also update the comment below re: kerning, as it currently only mentions the need to handle multibyte chars.